modules/gateway/nodes.go

```diff
@@ -18,6 +18,18 @@ var (
 	errPeerGenesisID = errors.New("peer has different genesis ID")
 )
 
+// A node represents a potential peer on the Sia network. The Prioritize field
+// indicates whether the node should be prioritized when forming new
+// connections.
+type node struct {
+	NetAddress modules.NetAddress `json:"netaddress"`
+	Prioritize bool               `json:"prioritize"`
```

Comment:
Ah yeah, Outbound is pretty vague. The real thing that's important is "we selected them randomly, but at least we know they were not explicitly chosen by an adversary. Also we are careful to keep the number of these selected peers very small."

I was thinking 'trusted', but that really indicates that the user specified them as a trusted peer explicitly. 

'Anchor' I think matches the literature on this stuff.